### DISCUSSION TITLE
Step by step or exact specification of the fat32 drive UMAF/UAFB difference?

### DISCUSSION THREAD

**Discussion by evlo** [DavidS95/Smokeless_UMAF#54] [General]
I have created non bootable fat32 drive in rufus and it does boot, what a shocker, so should the drive be:
gpt?
mbr?
bootable
bootable? with what? Ms dos? freedos?

target system is what? It only allows me to set bios (uefi-csm), there is no uefi option.

also my laptop only allows uefi boot, no legacy boot possible
![20240104_165039](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/b9c68646-791f-43e5-a7e2-2b4d02efe41b)
In the screenshot it does say dual or legacy, but non of the options exist
maybe I can't use this then?

Can you please update the specification of the usb drive for booting? 
And system requirements to make this work (ie. legacy boot (so called uefi csm]

![image](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/d00f36bc-25d8-442c-976f-cadc96dfff6f)

![image](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/35a0ec7c-eedb-4ea0-b283-5547ab27e37d)

I have also tried every .efi file in "boot from file" in the UEFI menu of the laptop (or whatever the thing with device manager etc. is) and again, nothing shows up, same is if I try to boot from usb.

What is the difference between `UMAF_BETA.zip` and `UniversalAMDFormBrowser.zip`? Readme.md only mentions UniversalAMDFormBrowser

Maybe the issue that mechrevo has InsydeH2O and most others do have AMI BIOS?

**Comment by evlo** [comment]
I managed to boot from the usb

windows > advanced boot options > throubleshoot > boot uefi > secure boot manager (or something like that)

if you do exactly this way, do not open anything before secure boot manager or open uefi menu any other way, you can access settings there, otherwise it gives error, otherwise it gives errors that it is not loaded as first thing after boot.



However I see no AMD PBS or AMD CBS options in the thing that boots up from the usb.

![image](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/25591c23-7aab-497c-b6db-b6603bf93c43)


first device manager menu option
![20240104_182125](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/0eb1f615-dafd-4834-a8ca-772677b4ddc8)

second device manager menu option
![20240104_182140](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/38a15008-e750-412b-8683-e910a9c44e57)

**Comment by evlo** [comment]
This is the best I could do in capturing boot log for `UMAF_BETA.zip`

I just took a photo of it. Is there a way to save the log or display it to take better photo?

![Screenshot_20240104_224817_Gallery](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/4d5f5cdd-2a48-4f48-8b67-b7f2fa890b29)



****


https://ruexe.blogspot.com/ no idea what to set
![20240107_042247](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/43b5cef0-2cb4-4597-920c-1f8c865158a8)
![20240107_043301](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/6ab0f891-cabc-4895-8d74-721b502cc7ed)
![20240107_043258](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/12ac3984-1950-429c-8ed5-da58a8e12c3f)
![20240107_043254](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/b29c86e8-2a0a-4c10-93a0-60366b2afb27)
![20240107_043250](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/c0fa84c7-2f69-4118-85f9-7110c5a99a30)
![20240107_043246](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/039b4321-9016-4b12-9180-71d7637e4c84)



https://gist.github.com/raenye/d6645d7039a6136ccfb055e0f8517698?permalink_comment_id=4394317#gistcomment-4394317

**Comment by evlo** [comment]
I'm about 6% sure this should be 00 to make s3 work
![image](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/24932439-44e5-4564-aa94-570c83cbc43c)
but do not want brick 

"14 14 01 03 01 ff 04 00"

**Comment by evlo** [comment]
as shown in screenshots above, in points
1. https://ruexe.blogspot.com/
2. edit amd pbs setup 
`14 14 01 03 01 ff 04 01`
`14 14 01 03 01 ff 04 00` **00** at the end
might be a bit different different but generally should point you to the correct place
![image](https://github.com/DavidS95/Smokeless_UMAF/assets/8813196/55f77360-dfd4-49f7-b128-5a1a6668c8e5)

**Comment by 37-37** [comment]
would you please make a step-by-step tutorial for noobs like me to follow?
